### DISCUSSION TITLE
Welcome to GDC Discussions!

### DISCUSSION THREAD

**Discussion by paulrobello** [devxpod/GDC#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mkjmkumar** [comment]
Hi Paulrobello,
  Thank you for your efforts to streamline complexities. My background involves deploying products on EKS using a microservices layered architecture. I managed to deploy the solution, focusing on the five pillars of AWS. However, I believe there are several areas that require attention to simplify integrations within this technological stack. Although the project with the client is completed, I retain that valuable experience. I'm keen on contributing towards streamlining integration challenges we face. I'm currently setting up GDC locally and exploring the iVTech blueprint use case to gain a better understanding of this tool. I would be delighted to offer further assistance in any other aspect of this project, if needed. 

Thanks & Regards,
Mukesh K.
